src/backend/efiling-api/src/main/resources/application.yml

```diff
@@ -28,6 +28,10 @@ jag:
           userName: ${CSO_LOOKUPFACADE_USERNAME}
           password: ${CSO_LOOKUPFACADE_PASSWORD}
           uri: ${CSO_LOOKUPFACADE_URI:http://localhost:8091}
+        - client: BCEID
+          userName: ${CSO_BCEIDLOOKUP_USERNAME}
```

Comment:
nit/question: should this be `CSO_BCEID_LOOKUP_USERNAME`?

Reply:
Could be, I was just keeping it consistent looking with the other clients

Reply:
Actually, I guess what it should be is BCEID_LOOKUP_USERNAME since it is technically not a CSO thing...

Reply:
yeah that last suggestion sounds great

Reply:
And it is done!